.install_dependency.sh

```diff
@@ -6,8 +6,9 @@
 # Codes taken from https://github.com/intel-ros/realsense/blob/c5ea27245967e0938f7d10384f4b7279e01000b4/.travis.yml
 # When used on CI sudo is not usually needed but it is most likely needed when used on your local computer.
 
-echo 'deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main' || sudo tee /etc/apt/sources.list.d/realsense-public.list
-add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main"
+COMMAND="deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo $(lsb_release -sc) main"
+echo $COMMAND | tee /etc/apt/sources.list.d/realsense-public.list
```

Comment:
With the current code CI can still pass because it just hits the [line that works](https://github.com/plusone-robotics/realsense/blob/f15d208825d2caf2a4516a000ca0a09d9c7040de/.install_dependency.sh#L9), but the [one line above](https://github.com/plusone-robotics/realsense/blob/f15d208825d2caf2a4516a000ca0a09d9c7040de/.install_dependency.sh#L8) is full of bug (which doesn't matter on CI since the code needs to run only once. But assuming this script can be used on non-CI environment, it's still worth fixed.